### PR TITLE
resolve MUWM-3856

### PR DESCRIPTION
**Pull request by fanglinfang**
test with bill100

### CODE REVIEW

**Comment by fanglinfang** [comment]
section links are also broken. hold the review until i fix that

File: myuw/static/js/card/schedule/visual.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var VisualScheduleCard = {
 
     should_display_card: function() {
         if (!window.user.student) {
+            return false;
```
Comment (mikeseibel) [inline myuw/static/js/card/schedule/visual.js]: what about the integrated instructor schedule?

Comment (fanglinfang) [inline myuw/static/js/card/schedule/visual.js]: good catch! will fix it

**Comment by bexar** [comment]
screen shot 2017-07-19 at 9 19 35 am

**Comment by fanglinfang** [comment]
@bexar   i have updated the page title. could you please update the style?

**Comment by bexar** [comment]
@fanglinfang I fixed the styles on course section card but noticed none of the grade submission information is displaying. 
screen shot 2017-07-21 at 10 58 31 am

**Comment by fanglinfang** [comment]
grade submission fix was in uw_sws. should should up by now.

**Comment by fanglinfang** [comment]
bill100

**Comment by klpittman** [comment]
myuw__home

@bexar is this the expected behavior? There's no links to specific classes or to a list of the classes scheduled for that quarter.

**Comment by klpittman** [comment]
myuw__teaching_section

Layout and styling look good. 

@bexar Just thinking, if a user goes to a specific section page, should we add the year/quarter info back somewhere? Without the tabs/menu from the Teaching page, it isn't easy to figure out what quarter you are looking at (it's in the URL, but that's about it). 

I dropped some potential locations I thought could work on the screen grab.

**Comment by klpittman** [comment]
@bexar I added the quarter/year where the date goes. We can reassess position/style later too.

**Comment by bexar** [comment]
@fanglinfang when i click on course from teaching page I get redirected to homepage, are you seeing the same thing?

**Comment by fanglinfang** [comment]
Yeah, i was reset to home page. will check into it

**Comment by fanglinfang** [comment]
Thanks much, Mike!

**Comment by klpittman** [comment]
Looks like @mikeseibel fixed it.

**Pull request merged**